modules/ROOT/pages/flex-gateway-modes.adoc

```diff
@@ -0,0 +1,25 @@
+= Flex Gateway Modes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Anypoint Flex Gateway can run in two modes:
+
+== Local Mode
+
+You can download Anypoint Flex Gateway for free and install it on a Linux server, VM, or Kubernetes cluster to immediately start applying out-of-the-box policies to non-Mule APIs running on those targets through declarative configuration files.
```

Comment:
I am not sure if the "for free" part is true. In any case I think it would be best to remove any references to pricing from the usage docs.

Reply:
reference removed